turborepo-tests/e2e/monorepo.ts

```diff
@@ -9,35 +9,55 @@ const turboPath = path.join(
   __dirname,
   "../../target/debug/turbo" + (isWin ? ".exe" : "")
 );
+import type { PackageManager } from "./types";
 
-type NPMClient = "npm" | "pnpm6" | "pnpm" | "yarn" | "berry";
+const PACKAGE_MANAGER_VERSIONS: { [key in PackageManager]: string } = {
+  yarn: "yarn@1.22.17",
+  berry: "yarn@3.1.1",
+  pnpm6: "pnpm@6.22.2",
+  pnpm: "pnpm@7.2.1",
+  npm: "npm@8.3.0",
+};
```

Comment:
I am interested in this piece. This is the current package managers we're validating with these tests, which is good to know. I'd be surprised if it matched the integration test suite

Reply:
We do have the _ability_ to use other package managers in our setup scripts:

https://github.com/vercel/turbo/blob/2b7ca782cce807231177b65c91ed00c61e796425/turborepo-tests/integration/README.md#L33-L37

But I don't see many examples. Here's one at least:
https://github.com/vercel/turbo/blob/2b7ca782cce807231177b65c91ed00c61e796425/turborepo-tests/integration/tests/single_package/run-yarn.t#L3